plugins/node/opentelemetry-instrumentation-restify/src/instrumentation.ts

```diff
@@ -37,11 +37,7 @@ import type { RestifyInstrumentationConfig } from './types';
 
 export class RestifyInstrumentation extends InstrumentationBase {
   constructor(config: RestifyInstrumentationConfig = {}) {
-    super(
-      `@opentelemetry/instrumentation-${constants.MODULE_NAME}`,
-      VERSION,
-      config
-    );
+    super(PACKAGE_NAME, PACKAGE_VERSION, config);
```

Comment:
same as router - As part of this change I think we could remove the `MODULE_NAME` constant from the `constants.ts` file. Then we can also rename in the `init()` there is another `constants.MODULE_NAME`.

Reply:
agree. will handle as follow-up PR